releasenotes/notes/adapt-vqe-0f71234cb6ec92f8.yaml

```diff
@@ -0,0 +1,27 @@
+---
+features:
+  - |
+    Implements the :class:`qiskit.algorithms.minimum_eigensolvers.AdaptVQE`
+    algorithm. This algorithm uses a
+    :class:`qiskit.algorithms.minimum_eigensolvers.VQE` in combination with a
+    pool of operators from which to build out an
+    :class:`qiskit.circuit.library.EvolvedOperatorAnsatz` adaptively.
+
+    .. code-block:: python
+
+      from qiskit.algorithms.minimum_eigensolvers import AdaptVQE, VQE
+      from qiskit.algorithms.optimizers import SLSQP
+      from qiskit.primitives import Estimator
+      from qiskit.circuit.library import EvolvedOperatorAnsatz
+
+      # get your Hamiltonian
+      hamiltonian = ...
+
+      # construct your ansatz
+      ansatz = EvolvedOperatorAnsatz(...)
+
+      vqe = VQE(Estimator(), ansatz, SLSQP())
+
+      adapt_vqe = AdaptVQE(vqe)
+
+      eigenvalue, _ = adapt_vqe.compute_minimum_eigenvalue(hamiltonian)
```

Comment:
This gives an AdaptVQE result back right of which the eigenvalue is accessed from the result.